itwin-platform-access/imodels-access-frontend/tsconfig.esm.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "./node_modules/@itwin/imodels-client-common-config/tsconfig.esm.json",
+  "compilerOptions": {
+    "outDir": "./lib/esm",
+    "esModuleInterop": true
```

Comment:
why is this in the config for @itwin/imodels-access-frontend and not for @itwin/imodels-client-management? maybe we could move this option to base config as it [is a recommended flag](https://www.typescriptlang.org/tsconfig#esModuleInterop)?

Reply:
esModuleInterop was specifically set for this package before. I think it can be safely removed from both configs, as there are no "import * as ..." style imports at all

Reply:
let's remove it if it's not needed then